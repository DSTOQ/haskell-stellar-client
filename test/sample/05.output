TransactionEnvelope
  { _transaction =
      Transaction
        { _sourceAccount =
            PublicKeyEd25519
              { _publicKeyEd25519 =
                  [ 97
                  , 51
                  , 50
                  , 55
                  , 53
                  , 49
                  , 57
                  , 56
                  , 51
                  , 51
                  , 55
                  , 48
                  , 57
                  , 55
                  , 55
                  , 57
                  , 99
                  , 99
                  , 51
                  , 57
                  , 98
                  , 54
                  , 52
                  , 51
                  , 102
                  , 55
                  , 57
                  , 97
                  , 52
                  , 101
                  , 97
                  , 51
                  , 51
                  , 50
                  , 97
                  , 56
                  , 98
                  , 56
                  , 55
                  , 100
                  , 100
                  , 54
                  , 49
                  , 49
                  , 49
                  , 48
                  , 51
                  , 53
                  , 51
                  , 49
                  , 57
                  , 48
                  , 57
                  , 55
                  , 50
                  , 53
                  , 51
                  , 54
                  , 51
                  , 97
                  , 57
                  , 53
                  , 101
                  , 100
                  ]
              }
        , _fee = Fee { _fee = 500 }
        , _seqNum = SequenceNumber { _sequenceNumber = 46283168877117507 }
        , _timeBounds = Nothing
        , _memo = MemoNone
        , _operations =
            [ Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling =
                            AssetCreditAlphanum
                              AssetCode { _assetCode = "CNY" }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    [ 49
                                    , 97
                                    , 57
                                    , 51
                                    , 50
                                    , 52
                                    , 50
                                    , 54
                                    , 53
                                    , 56
                                    , 97
                                    , 99
                                    , 102
                                    , 56
                                    , 49
                                    , 56
                                    , 101
                                    , 49
                                    , 49
                                    , 99
                                    , 98
                                    , 97
                                    , 48
                                    , 56
                                    , 51
                                    , 55
                                    , 53
                                    , 56
                                    , 52
                                    , 102
                                    , 52
                                    , 55
                                    , 53
                                    , 55
                                    , 57
                                    , 50
                                    , 50
                                    , 53
                                    , 52
                                    , 49
                                    , 97
                                    , 55
                                    , 54
                                    , 55
                                    , 55
                                    , 101
                                    , 57
                                    , 57
                                    , 50
                                    , 98
                                    , 50
                                    , 49
                                    , 50
                                    , 49
                                    , 55
                                    , 54
                                    , 97
                                    , 49
                                    , 57
                                    , 101
                                    , 99
                                    , 48
                                    , 101
                                    , 51
                                    ]
                                }
                        , _buying = AssetNative
                        , _amount = 80000000
                        , _price = Price { _numerator = 2271 , _denominator = 10000 }
                        , _offerId = OfferId { _offerId = 0 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling = AssetNative
                        , _buying =
                            AssetCreditAlphanum
                              AssetCode { _assetCode = "USD" }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    [ 49
                                    , 97
                                    , 57
                                    , 51
                                    , 50
                                    , 52
                                    , 50
                                    , 54
                                    , 53
                                    , 56
                                    , 97
                                    , 99
                                    , 102
                                    , 56
                                    , 49
                                    , 56
                                    , 101
                                    , 49
                                    , 49
                                    , 99
                                    , 98
                                    , 97
                                    , 48
                                    , 56
                                    , 51
                                    , 55
                                    , 53
                                    , 56
                                    , 52
                                    , 102
                                    , 52
                                    , 55
                                    , 53
                                    , 55
                                    , 57
                                    , 50
                                    , 50
                                    , 53
                                    , 52
                                    , 49
                                    , 97
                                    , 55
                                    , 54
                                    , 55
                                    , 55
                                    , 101
                                    , 57
                                    , 57
                                    , 50
                                    , 98
                                    , 50
                                    , 49
                                    , 50
                                    , 49
                                    , 55
                                    , 54
                                    , 97
                                    , 49
                                    , 57
                                    , 101
                                    , 99
                                    , 48
                                    , 101
                                    , 51
                                    ]
                                }
                        , _amount = 60000000
                        , _price = Price { _numerator = 2099 , _denominator = 10000 }
                        , _offerId = OfferId { _offerId = 0 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling =
                            AssetCreditAlphanum
                              AssetCode { _assetCode = "XRP" }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    [ 49
                                    , 97
                                    , 57
                                    , 51
                                    , 50
                                    , 52
                                    , 50
                                    , 54
                                    , 53
                                    , 56
                                    , 97
                                    , 99
                                    , 102
                                    , 56
                                    , 49
                                    , 56
                                    , 101
                                    , 49
                                    , 49
                                    , 99
                                    , 98
                                    , 97
                                    , 48
                                    , 56
                                    , 51
                                    , 55
                                    , 53
                                    , 56
                                    , 52
                                    , 102
                                    , 52
                                    , 55
                                    , 53
                                    , 55
                                    , 57
                                    , 50
                                    , 50
                                    , 53
                                    , 52
                                    , 49
                                    , 97
                                    , 55
                                    , 54
                                    , 55
                                    , 55
                                    , 101
                                    , 57
                                    , 57
                                    , 50
                                    , 98
                                    , 50
                                    , 49
                                    , 50
                                    , 49
                                    , 55
                                    , 54
                                    , 97
                                    , 49
                                    , 57
                                    , 101
                                    , 99
                                    , 48
                                    , 101
                                    , 51
                                    ]
                                }
                        , _buying = AssetNative
                        , _amount = 60000000
                        , _price = Price { _numerator = 1307 , _denominator = 5000 }
                        , _offerId = OfferId { _offerId = 0 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling = AssetNative
                        , _buying =
                            AssetCreditAlphanum
                              AssetCode { _assetCode = "XYG" }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    [ 49
                                    , 97
                                    , 57
                                    , 51
                                    , 50
                                    , 52
                                    , 50
                                    , 54
                                    , 53
                                    , 56
                                    , 97
                                    , 99
                                    , 102
                                    , 56
                                    , 49
                                    , 56
                                    , 101
                                    , 49
                                    , 49
                                    , 99
                                    , 98
                                    , 97
                                    , 48
                                    , 56
                                    , 51
                                    , 55
                                    , 53
                                    , 56
                                    , 52
                                    , 102
                                    , 52
                                    , 55
                                    , 53
                                    , 55
                                    , 57
                                    , 50
                                    , 50
                                    , 53
                                    , 52
                                    , 49
                                    , 97
                                    , 55
                                    , 54
                                    , 55
                                    , 55
                                    , 101
                                    , 57
                                    , 57
                                    , 50
                                    , 98
                                    , 50
                                    , 49
                                    , 50
                                    , 49
                                    , 55
                                    , 54
                                    , 97
                                    , 49
                                    , 57
                                    , 101
                                    , 99
                                    , 48
                                    , 101
                                    , 51
                                    ]
                                }
                        , _amount = 10000000
                        , _price = Price { _numerator = 73 , _denominator = 250 }
                        , _offerId = OfferId { _offerId = 0 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling = AssetNative
                        , _buying =
                            AssetCreditAlphanum
                              AssetCode { _assetCode = "XLM" }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    [ 49
                                    , 97
                                    , 57
                                    , 51
                                    , 50
                                    , 52
                                    , 50
                                    , 54
                                    , 53
                                    , 56
                                    , 97
                                    , 99
                                    , 102
                                    , 56
                                    , 49
                                    , 56
                                    , 101
                                    , 49
                                    , 49
                                    , 99
                                    , 98
                                    , 97
                                    , 48
                                    , 56
                                    , 51
                                    , 55
                                    , 53
                                    , 56
                                    , 52
                                    , 102
                                    , 52
                                    , 55
                                    , 53
                                    , 55
                                    , 57
                                    , 50
                                    , 50
                                    , 53
                                    , 52
                                    , 49
                                    , 97
                                    , 55
                                    , 54
                                    , 55
                                    , 55
                                    , 101
                                    , 57
                                    , 57
                                    , 50
                                    , 98
                                    , 50
                                    , 49
                                    , 50
                                    , 49
                                    , 55
                                    , 54
                                    , 97
                                    , 49
                                    , 57
                                    , 101
                                    , 99
                                    , 48
                                    , 101
                                    , 51
                                    ]
                                }
                        , _amount = 80000000
                        , _price = Price { _numerator = 303 , _denominator = 1000 }
                        , _offerId = OfferId { _offerId = 0 }
                        }
                }
            ]
        }
  , _signatures =
      [ DecoratedSignature
          { _hint = SignatureHint [ 51 , 54 , 51 , 97 , 57 , 53 , 101 , 100 ]
          , _signature =
              Signature
                { _signature =
                    "[55,53,97,57,54,49,56,54,55,99,98,55,55,52,49,101,50,57,101,100,50,52,48,100,101,56,98,98,57,48,101,102,52,57,50,57,100,49,53,101,98,53,48,98,102,54,55,53,49,48,102,51,102,48,57,48,54,101,98,51,57,49,56,52,97,100,56,101,52,49,50,98,98,48,50,55,50,97,98,52,102,56,98,49,57,101,97,48,55,55,51,54,55,48,56,52,55,50,57,50,57,50,100,101,99,100,50,53,102,54,55,98,48,99,55,48,54,55,100,55,48,49,99,97,53,56,48,98]"
                }
          }
      ]
  }