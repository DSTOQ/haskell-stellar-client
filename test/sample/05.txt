AAAAAKMnUZgzcJd5zDm2Q/eaTqMyqLh91hEQNTGQlyU2OpXtAAAB9ACkbkwAAABDAAAAAAAAAAAAAAAFAAAAAAAAAAMAAAABQ05ZAAAAAAAakyQmWKz4GOEcugg3WE9HV5IlQadnfpkrISF2oZ7A4wAAAAAAAAAABMS0AAAACN8AACcQAAAAAAAAAAAAAAAAAAAAAwAAAAAAAAABVVNEAAAAAAAakyQmWKz4GOEcugg3WE9HV5IlQadnfpkrISF2oZ7A4wAAAAADk4cAAAAIMwAAJxAAAAAAAAAAAAAAAAAAAAADAAAAAVhSUAAAAAAAGpMkJlis+BjhHLoIN1hPR1eSJUGnZ36ZKyEhdqGewOMAAAAAAAAAAAOThwAAAAUbAAATiAAAAAAAAAAAAAAAAAAAAAMAAAAAAAAAAVhZRwAAAAAAGpMkJlis+BjhHLoIN1hPR1eSJUGnZ36ZKyEhdqGewOMAAAAAAJiWgAAAAEkAAAD6AAAAAAAAAAAAAAAAAAAAAwAAAAAAAAABWExNAAAAAAAakyQmWKz4GOEcugg3WE9HV5IlQadnfpkrISF2oZ7A4wAAAAAExLQAAAABLwAAA+gAAAAAAAAAAAAAAAAAAAABNjqV7QAAAEB1qWGGfLd0HintJA3ou5DvSSnRXrUL9nUQ8/CQbrORhK2OQSuwJyq0+LGeoHc2cIRykpLezSX2ewxwZ9cBylgL
