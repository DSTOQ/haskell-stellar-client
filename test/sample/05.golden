TransactionEnvelope
  { transaction =
      Transaction
        { sourceAccount =
            PublicKeyEd25519
              { unPublicKeyEd25519 =
                  73796464499382006151152614252603807196781827224650109053774855681839325681133
              }
        , fee = Fee { unFee = 500 }
        , seqNum = SequenceNumber { unSequenceNumber = 46283168877117507 }
        , timeBounds = Nothing
        , memo = MemoNone
        , operations =
            VarLen
              { unVarLen =
                  [ Operation
                      { sourceAccount = Nothing
                      , body =
                          ManageOffer
                            ManageOfferOp
                              { selling =
                                  AssetCreditAlphanum4
                                    AssetCode4 { unAssetCode4 = 1129208064 }
                                    PublicKeyEd25519
                                      { unPublicKeyEd25519 =
                                          12020110078372443445778830331393468099140010802731689699023048901958341411043
                                      }
                              , buying = AssetNative
                              , amount = 80000000
                              , price = Price { numerator = 2271 , denominator = 10000 }
                              , offerId = OfferId { unOfferId = 0 }
                              }
                      }
                  , Operation
                      { sourceAccount = Nothing
                      , body =
                          ManageOffer
                            ManageOfferOp
                              { selling = AssetNative
                              , buying =
                                  AssetCreditAlphanum4
                                    AssetCode4 { unAssetCode4 = 1431520256 }
                                    PublicKeyEd25519
                                      { unPublicKeyEd25519 =
                                          12020110078372443445778830331393468099140010802731689699023048901958341411043
                                      }
                              , amount = 60000000
                              , price = Price { numerator = 2099 , denominator = 10000 }
                              , offerId = OfferId { unOfferId = 0 }
                              }
                      }
                  , Operation
                      { sourceAccount = Nothing
                      , body =
                          ManageOffer
                            ManageOfferOp
                              { selling =
                                  AssetCreditAlphanum4
                                    AssetCode4 { unAssetCode4 = 1481789440 }
                                    PublicKeyEd25519
                                      { unPublicKeyEd25519 =
                                          12020110078372443445778830331393468099140010802731689699023048901958341411043
                                      }
                              , buying = AssetNative
                              , amount = 60000000
                              , price = Price { numerator = 1307 , denominator = 5000 }
                              , offerId = OfferId { unOfferId = 0 }
                              }
                      }
                  , Operation
                      { sourceAccount = Nothing
                      , body =
                          ManageOffer
                            ManageOfferOp
                              { selling = AssetNative
                              , buying =
                                  AssetCreditAlphanum4
                                    AssetCode4 { unAssetCode4 = 1482245888 }
                                    PublicKeyEd25519
                                      { unPublicKeyEd25519 =
                                          12020110078372443445778830331393468099140010802731689699023048901958341411043
                                      }
                              , amount = 10000000
                              , price = Price { numerator = 73 , denominator = 250 }
                              , offerId = OfferId { unOfferId = 0 }
                              }
                      }
                  , Operation
                      { sourceAccount = Nothing
                      , body =
                          ManageOffer
                            ManageOfferOp
                              { selling = AssetNative
                              , buying =
                                  AssetCreditAlphanum4
                                    AssetCode4 { unAssetCode4 = 1481395456 }
                                    PublicKeyEd25519
                                      { unPublicKeyEd25519 =
                                          12020110078372443445778830331393468099140010802731689699023048901958341411043
                                      }
                              , amount = 80000000
                              , price = Price { numerator = 303 , denominator = 1000 }
                              , offerId = OfferId { unOfferId = 0 }
                              }
                      }
                  ]
              }
        }
  , signatures =
      VarLen
        { unVarLen =
            [ DecoratedSignature
                { hint = SignatureHint { unSignatureHint = 909809133 }
                , signature =
                    Signature
                      { unSignature =
                          VarLen
                            { unVarLen =
                                "u\169a\134|\183t\RS)\237$\r\232\187\144\239I)\209^\181\v\246u\DLE\243\240\144n\179\145\132\173\142A+\176'*\180\248\177\158\160w6p\132r\146\146\222\205%\246{\fpg\215\SOH\202X\v"
                            }
                      }
                }
            ]
        }
  }